dcc-portal-ui/Gruntfile.js

```diff
@@ -483,16 +483,33 @@ module.exports = function (grunt) {
           ignorePath: 'app'
         },
         files: [{
-          expand: true,
-          cwd: '<%= yeoman.app %>/develop/scripts',
-          src: ['*.js']
-        }]
+            expand: true,
+            cwd: '<%= yeoman.app %>/develop/scripts',
+            src: ['*.js']
+          },
+          {
+            expand: true,
+            cwd: '<%= yeoman.app %>/branding/scripts',
+            src: ['*.js']
+          }
+        ]
       }
     }
   });
 
   grunt.registerTask('bower-install', ['bower-install-simple']);
 
+  // A very basic task to read from process.env
+  grunt.registerTask('branding_customize', 'Override UI images and strings. see app/branding/scripts/config.js', function() {
+    if (process.env.ICGC_BRANDING) {
+      var _fileName = grunt.template.process('<%= yeoman.app %>/branding/scripts/custom.js');
+      grunt.file.write(_fileName, '$ICGC_BRANDING = ' + process.env.ICGC_BRANDING);
```

Comment:
Could we have the BRANDING variable be a file name rather than containing the JSON?  It likely that when this is build, the variable will be set in a script, so the JSON would be in a file anyway.  It would be clearer to have the branding JSON in its own file rather than in a variable, I think.